run_composeEff_rooKeys.sh

```diff
@@ -22,6 +21,11 @@ ybin=${8}
 zbin=${9}
 ndiv=${10}
 totdiv=${11}
+year=${12}
+
+export SAMPLEDIR=/eos/cms/store/user/fiorendi/p5prime/effKDE/${year}/lmnr/
+
+echo 'now submitting for bin ' ${bin}
 
 if [ ! -r $SAMPLEDIR/effDataset_b${bin}.root ]; then
     echo $SAMPLEDIR/effDataset_b${bin}.root not found
```

Comment:
The filename should be updated to `$SAMPLEDIR/effDataset_b${bin}_${year}.root`

Reply:
solved